svg3d.py

```diff
@@ -132,7 +132,7 @@ def from_string(cls, string_to_parse: str):
 class Mesh:
     def __init__(
         self,
-        poly: "coxeter.shapes.ConvexPolyhedron" | None = None,
+        poly: str | None = None,  # noqa: F821
```

Comment:
```suggestion
        poly: "coxeter.shapes.ConvexPolyhedron" | None = None,  # noqa: F821
```

TODO: update typing requirements